docs/Performance and Tuning/Workload Tuning.rst

```diff
@@ -50,7 +50,10 @@ zfs properties respectively, which can be set on both zvols and
 datasets. Possible settings are ``all``, ``none`` and ``metadata``. It
 is possible to improve performance when a zvol or dataset hosts an
 application that does its own caching by caching only metadata. One
-example is PostgreSQL. Another would be a virtual machine using ZFS.
+example would be a virtual machine using ZFS. Another would be a 
+database system which manages its own cache (Oracle for instance).
+PostgreSQL, by contrast, depends on the OS-level file cache for the 
```

Comment:
It would be great to have a source here too, or benchmark.

Reply:
Nearly any source regarding PostgreSQL tuning. Start with the documentation, which makes a brief reference to it: https://www.postgresql.org/docs/15/runtime-config-resource.html. 

Also:

https://distributedsystemsauthority.com/optimizing-postgresql-shared-buffers/
https://postgreshelp.com/postgresql_shared_buffers/
https://wiki.postgresql.org/wiki/Tuning_Your_PostgreSQL_Server
https://redfin.engineering/how-to-boost-postgresql-cache-performance-8db383dc2d8f
https://www.postgresql.org/message-id/20020515112447.C23717%40mail.libertyrms.com
https://www.postgresql.org/message-id/15338.1006185597@sss.pgh.pa.us
https://www.crunchydata.com/blog/optimize-postgresql-server-performance
https://severalnines.com/blog/architecture-and-tuning-memory-postgresql-databases/
https://bun.uptrace.dev/postgres/performance-tuning.html#shared-buffers
https://momjian.us/main/writings/pgsql/hw_performance/
https://gitlab.com/gitlab-org/gitlab/-/issues/258640 (see comments about eviction algo)

Reply:
LGTM